src/oas3/transformers/content.ts

```diff
@@ -110,13 +110,19 @@ export function translateMediaTypeObject(mediaObject: unknown, mediaType: string
 
   const { schema, encoding } = mediaObject;
 
-  const jsonSchema = schema
-    ? (toJsonSchema(schema, {
+  let jsonSchema: Optional<JSONSchema4>;
+
+  if (schema) {
+    try {
+      jsonSchema = toJsonSchema(schema, {
         cloneSchema: true,
         strictMode: false,
         keepNotSupported: ['example', 'deprecated', 'readOnly', 'writeOnly', 'xml', 'externalDocs'],
-      }) as JSONSchema4)
-    : undefined;
+      }) as JSONSchema4;
+    } catch {
+      // happens
```

Comment:
shall we report it somehow so we have a clue that XYZ thing in the convert fails and we might need to do something about it?

Reply:
We talked about this for ages. There's no way to log stuff from HTTP Spec, so we'll just ignore it :(